pkg/options/options.go

```diff
@@ -140,6 +142,8 @@ func (s *KubeRouterConfig) AddFlags(fs *pflag.FlagSet) {
 		"Enables the BGP Graceful Restart capability so that routes are preserved on unexpected restarts")
 	fs.DurationVar(&s.BGPGracefulRestartDeferralTime, "bgp-graceful-restart-deferral-time", s.BGPGracefulRestartDeferralTime,
 		"BGP Graceful restart deferral time according to RFC4724 4.1, maximum 18h.")
+	fs.Float64Var(&s.BGPHoldtime, "bgp-holdtime", DEFAULT_HOLDTIME,
```

Comment:
can you make this DEFAULT_BGP_HOLDTIME instead of DEFAULT_HOLDTIME (so it matches the other env vars)?

Reply:
Yes, of course. I have now set DEFAULT_HOLDTIME changed to DEFAULT_BGP_HOLDTIME。

Here are the relevant screenshots:
![20200612123745](https://user-images.githubusercontent.com/62274882/84465602-05cd9b80-acaa-11ea-9b03-8e42d18713ad.jpg)
![20200612123800](https://user-images.githubusercontent.com/62274882/84465608-09f9b900-acaa-11ea-8165-c5aa67230641.jpg)